app/assets/javascripts/alchemy/alchemy.sitemap.js.coffee

```diff
@@ -8,7 +8,7 @@ Alchemy.Sitemap =
     @search_field = $("#search_field")
     @filter_field_clear = $('.js_filter_field_clear')
     @display = $('#page_filter_result')
-    @sitemap_wrapper = $('#sitemap-wrapper .sitemap-load-notice')
+    @sitemap_wrapper = $('#sitemap-wrapper')
```

Comment:
The code replaces that target with the sitemap. There's some CSS attached to `#sitemap-wrapper`. This will break that. Instead, there should be a dummy element within the wrapper which can be safely replaced with the sitemap itself.